attributes/default.rb

```diff
@@ -5,7 +5,7 @@
 
 default['hops']['versions']                    = "2.8.2.2,2.8.2.3,2.8.2.4,2.8.2.5,2.8.2.6,2.8.2.7,2.8.2.8,2.8.2.9,2.8.2.10,3.2.0.0,3.2.0.1,3.2.0.2,3.2.0.3,3.2.0.4"
 default['hops']['version']                     = "3.2.0.5-SNAPSHOT"
-default['hops']['fuse']['version']             = "1.1.0"
+default['hops']['fuse']['version']             = "1.3.0"
```

Comment:
There was also version 1.2.0 . I forgot to update hops-hadoop-chef for version 1.2.0.
now version 1.3.0 is the latest.